DESCRIPTION

```diff
@@ -35,6 +35,7 @@ Imports:
     utils
 Suggests:
     cli (>= 3.4.0),
+    data.table (>= 1.16.2),
```

Comment:
`data.table` is only added so that we can execute one test (added in this PR). 
We can create a test without `data.table` but we would test internals

Below would need to be used in the test. `split_code` and `normalize_pd` and `extract_calls` are internal functions.

```r
code <- "
iris <- data.table::data.table(iris) %>%
  .[, NewSpecies := factor(Species)]
"

code_split <- split_code(paste(code, collapse = "\n"))[[1]]
current_call <- parse(text = code_split, keep.source = TRUE)
pd <- normalize_pd(utils::getParseData(current_call))
reordered_pd <- extract_calls(pd)
reordered_pd[[1]]
``

Reply:
Could we achieve the same with rlang alone? (as dynamic dots traces back to rlang)

If not with `list2` with some other function in `rlang` package

```r
x <- "name"
rlang::list2(!!x := 1)
```

Reply:
Thanks @averissimo will try with `rlang`! Thanks for the recommendation!

Reply:
Thanks, changed to rlang
https://github.com/insightsengineering/teal.code/pull/234/commits/6d38745e083fb668108850da3dbac92a18c3b747